test/cli_crud_test.go

```diff
@@ -136,7 +136,8 @@ func TestFnUpdateValues(t *testing.T) {
 	}
 
 	invalidCases := [][]string{
-		{"--image", "fooimage:1.0.0"}, // image with no registry
+		// image with no registry is valid case for local development
+		// {"--image", "fooimage:1.0.0"}, // image with no registry
```

Comment:
@denismakogon any chance to add actual test here (above) https://github.com/fnproject/cli/pull/562/files#diff-8e679447ea0f37d6dcbf0abaaa9c5f36L119 ?

Reply:
see #564